tst/EnergyPlus/unit/AirTerminalSingleDuct.unit.cc

```diff
@@ -1008,6 +1008,7 @@ TEST_F(EnergyPlusFixture, SingleDuctVAVReheatAirTerminal_MinFlowTurnDownTest)
     });
 
     ASSERT_TRUE(process_idf(idf_objects));
+    compare_err_stream_substring("", true); // clear idf errors
```

Comment:
Hmm, unexpected addition here.  Did new errors get emitted that you need to clear away?  It's not a big deal.  If we need a worker function over on the text fixture to clear away the err buffer, we could add one pretty easily:

```c++
void EnergyPlusFixture::clear_err_stream();  // in EnergyPlusFixture.hh
void EnergyPlusFixture::clear_err_stream()   // in EnergyPlusFixture.cc
{
    this->err_stream->str(std::string());
}
```

Then out here it would just be `clear_err_stream();`

Reply:
The move of GetProjectData to init_state caused additional text to appear ("Timestep < 4" was 1 of them). I was testing ways to avoid changing the unit test. I think I could have added the *substring method directly in the unit test instead of doing this or what you suggest.